CHANGELOG.md

```
@@ -0,0 +1,187 @@
+
+## [Unreleased] — 0.7.2
+
+| Commit | Change |
+|--------|--------|
+| `2caf21b` | feat(lab-h5pm.4): dispatch sync action with RAII AtomicBool rate-limit guard |
+| `8233ac5` | feat(registry): use GitHub owner avatar as server image |
+| `0d1acba` | feat(gateway-admin): aurora token sweep + eslint enforcement |
+| `04a0dbd` | feat(lab-h5pm.2): implement RegistryStore query methods, upsert, and full sync |
+| `96ddf66` | feat(lab-h5pm.1): create RegistryStore module skeleton in dispatch layer |
+
+### Highlights
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)